src/modules/tools/zprobe/ZProbe.cpp

```diff
@@ -176,15 +178,14 @@ bool ZProbe::run_probe(int& steps, bool fast)
 
     // Enable the motors
     THEKERNEL->stepper->turn_enable_pins_on();
-    this->current_feedrate = (fast ? this->fast_feedrate : this->slow_feedrate) * Z_STEPS_PER_MM; // steps/sec
-    float maxz= this->max_z*2;
```

Comment:
This was done for a reason, so should not be changed. an alternative is to set a maxz independent config for zprobe and not use the endstop maxz